setup.py

```diff
@@ -76,7 +76,7 @@ def run(self):
     scripts=['bin/dswx_hls.py',
     	     'bin/dswx_compare.py'],
     install_requires=['argparse', 'numpy', 'yamale',
-                      'osgeo', 'scipy', 'pytest', 'requests'],
+                      'scipy', 'pytest', 'requests'],
```

Comment:
`osgeo` contains access to `gdal`, which is a necessary dependency for PROTEUS.

If this dependency is removed, and then a user tries to install and run PROTEUS in a fresh conda environment, then it will fail due to the missing dependency.

If this fix worked on your system and you were still able to run PROTEUS, by chance, were you working in an existing conda environment, where `osgeo` / `gdal` were previously installed for a different application?

Reply:
> `osgeo` contains access to `gdal`, which is a necessary dependency for PROTEUS.
> 
> If this dependency is removed, and then a user tries to install and run PROTEUS in a fresh conda environment, then it will fail due to the missing dependency.
> 
> If this fix worked on your system and you were still able to run PROTEUS, by chance, were you working in an existing conda environment, where `osgeo` / `gdal` were previously installed for a different application?

`osgeo` is not the proper name for the library from what I have read and tried locally: https://pypi.org/project/osgeo/

Reply:
I tried adding `gdal` but now it's not passing CircleCI - I will let you figure out how to fix this or maybe it's not necessary.

Reply:
That doesn't surprise me that it failed. Please see this comment: https://github.com/nasa/PROTEUS/issues/51#issuecomment-1861629813

Updating PROTEUS to be installed via `pip` and not `setup.py` would likely be considered a new feature development.